pubpub/pubpub
# fix: allow null backgroundImage and 'none' button

## Issue(s) Resolved

## Test Plan

1. Go to HDSR
2. Open network tab
3. Try to modify homepage
4. Should not cause a problem.

## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas
Two issues

1. banner button with no text is "none", not `undefined`
2. allow backgroundImage to be `null`


### Supporting Docs
